examples/gallery/demos/bokeh/nyc_radial_heatmap.ipynb

```diff
@@ -0,0 +1,128 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load example data\n",
+    "df_nyc = pd.read_csv(\"../../../assets/nyc_taxi.csv.gz\", parse_dates=[\"Pickup_date\"])\n",
+    "\n",
+    "# create relevant time columns\n",
+    "df_nyc[\"Day & Hour\"] = df_nyc[\"Pickup_date\"].dt.strftime(\"%A %H:00\")\n",
+    "df_nyc[\"Week of Year\"] = df_nyc[\"Pickup_date\"].dt.strftime(\"Week %W\")\n",
+    "df_nyc[\"Date\"] = df_nyc[\"Pickup_date\"].dt.strftime(\"%Y-%m-%d\")\n",
+    "\n",
+    "heatmap = hv.HeatMap(df_nyc, [\"Day & Hour\", \"Week of Year\"], [\"Pickup_Count\", \"Date\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**At first glance**: First, let's take a closer look at the mentioned segments and annulars. **Segments** correspond to *hours of a given day* whereas **annulars** represent entire *weeks*. If you use the hover tool, you will quickly get an idea of how segments and annulars are organized. **Color** decodes the pickup values with blue being low and red being high.\n",
+    "\n",
+    "**Plot improvements**: The above plot clearly shows systematic patterns however the default plot options are somewhat disadvantageous. Therefore, before we start to dive into the results, let's increase the readability of the given plot:\n",
+    "\n",
+    "- **Remove annular ticks**: The information about week of year is not very important. Therefore, we hide it via `yticks=None`.\n",
+    "- **Custom segment ticks**: Right now, segment labels are given via day and hour. We don't need hourly information and we want every day to be labeled. We can use a tuple here which will be passed to `xticks=(\"Friday\", ..., \"Thursday\")`\n",
+    "- **Add segment markers**: Moreover, we want to aid the viewer in distingushing each day more clearly. Hence, we can provide marker lines via `xmarks=7`.\n",
+    "- **Rotate heatmap**: The week starts with Monday and ends with Sunday. Accordingly, we want to rotate the plot to have Sunday and Monday be at the top. This can be done via `start_angle=np.pi*19/14`. The default order is defined by the global sort order which is present in the data. The default starting angle is at 12 o'clock.\n",
+    "\n",
+    "Let's see the result of these modifications:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts HeatMap [radial=True width=600 height=600 yticks=None xmarks=7 ymarks=3 start_angle=np.pi*19/14]\n",
```

Comment:
Instead of having both ``radial=True`` (just a boolean) why not require ``start_angle`` which toggle radial if non-None? I would prefer to reduce the number of plot options and parameters involving ``angle`` only make sense to me for the radial case...

Reply:
Looking at the plotting class definition there seem to be quite a few parameters specific to radial. I wonder if the ``is_heatmap`` property could detect the use of radial heatmap parameters. The problem then is you would probably need the defaults to be ``None``...

Reply:
Much prefer being explicit about radial, anything else is confusing especially enabling it via the angle. I do agree that having options on HeatMap that only apply to the radial plot isn't ideal though.

Reply:
I suppose I don't mind the ``radial`` parameter working and existing as a boolean - if you are happy with all the other defaults for radial heatmaps, you can use it. But then if you do want to set a parameter that does clearly apply only to radial heatmaps, it becomes unnecessary. This logic would probably all be in ``is_radial``.

Not sure it is worth the effort though...

Reply:
Seems reasonable, let me think about it. Maybe I'll try to reduce the number of parameters that are exclusive to radial heatmaps first, the xmarks and ymarks parameters for instance could be ported to the regular heatmap.

Reply:
That was on my mind as another comment though I didn't bring it up. Glad you mentioned it as I agree!